code/modules/scripting/Implementations/Telecomms.dm

```diff
@@ -244,7 +244,7 @@ var/list/allowed_custom_spans = list(SPAN_ROBOT,SPAN_YELL,SPAN_ITALICS,SPAN_SANS
 		signal.data["verb_yell"]	= interpreter.GetCleanVar("$yell")
 		signal.data["verb_exclaim"]	= interpreter.GetCleanVar("$exclaim")
 		var/list/setspans 			= interpreter.GetCleanVar("$filters") //Save the span vector/list to a holder list
-		setspans &= allowed_custom_spans //Prune out any illegal ones. Go ahead, comment this line out. See the horror you can unleash!
+//		setspans &= allowed_custom_spans //Prune out any illegal ones. Go ahead, comment this line out. See the horror you can unleash!  //Release the SPANS!
 		if(islist(setspans)) //Previous comment block was right. Players cannot be trusted with ANYTHING. At all. Ever.
```

Comment:
You may wish to move this line ABOVE the illegal spans checker, else you may have runtimes for users that do not properly put the spans in vectors. May also need a check for non-string values in the list. I forgot to do this for /tg/, but it does not matter now, as NTSL was removed from /tg/ because MrPerson hates anything fun.